tests/phpunit/includes/Core/Modules/FakeModule.php

```diff
@@ -108,9 +108,9 @@ public function register() {
 	 * @since 1.0.0
```

Comment:
This doc block should be updated as it still has the same one used for datapoint services which is not completely accurate.

Reply:
Updated.